pygfx/utils/transform.py

```diff
@@ -20,12 +20,14 @@ class cached:  # noqa: N801
 
     """
 
+    __slots__ = ("compute_fn", "name")
+
     def __init__(self, compute_fn=None) -> None:
         self.compute_fn = compute_fn
         self.name = None
 
     def __set_name__(self, clazz, name) -> None:
-        self.name = f"_{name}_cache"
+        self.name = f"cache_{name}_cache"
```

Comment:
Is this intentional (the name "cache" occuring twice)?

Reply:
* Yes, it doesn't really matter what it is, as long as it's unlikely to clash with attribute names actually want to use.
* My problem with the original name was that it led to double underscores if `name` happened to be prefixed with a single underscore, and that in turn triggered name mangling. That's why I put some words in the prefix here.